ansible/roles/test/files/ptftests/advanced-reboot.py

```diff
@@ -275,8 +279,22 @@ def read_json(self, name):
 
     def read_port_indices(self):
         port_indices = self.read_json('ports_file')
+        if self.test_params['is_dualtor']:
+            peer_port_indices = self.read_json('peer_ports_file')
+            return port_indices, peer_port_indices
```

Comment:
Minor:
Not ideal that a method  returns two different types - eg., here - list and tuple (containing lists).

Can you normalize to same (tuple?), and let the caller handle the return based on is_dualtor

Reply:
Updated